server/app_data/settings_app_data.yaml.example

```diff
@@ -0,0 +1,2 @@
+---
+site_hardware_id: ""
```

Comment:
I thought that the yaml file was an internally managed store? In which case it won't require an example, as no one will be editing it manually